en/orm/query-builder.rst

```diff
@@ -1181,6 +1181,16 @@ subqueries::
 Subqueries are accepted anywhere a query expression can be used. For example, in
 the ``select()`` and ``join()`` methods.
 
+Executing Complex Queries
+-------------------------
+
+While the query builder makes it easy to build most queries, very complex
+queries can be tedious and complicated to build. You may want to :ref:`execute
+the desired SQL directly <running-select-statements>`.
+
+Executing SQL allows you to fine tune the query that will be run, but doesn't
```

Comment:
> Executing SQL...

Something is missing :confused: 
Like the sentense or its meaning is not complete.